pixeltable/iterators/document.py

```diff
@@ -213,12 +213,6 @@ def output_schema(cls, *args: Any, **kwargs: Any) -> tuple[dict[str, ColumnType]
             if kwargs.get('limit') is None:
                 raise Error('limit is required with "token_limit"/"char_limit" separators')
 
-        # check dependencies at the end
```

Comment:
So where is the importerror showing up now?

Reply:
When you insert new data or otherwise cause the iterator to be invoked.